fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -437,6 +438,11 @@ def exec_primary_data_func():
             self.__events_loading or self.__users_loading or self.__groups_loading
         )
 
+        if system.get("changesets"):
+            if not self.__n_changesets == len(system.get("changesets")):
+                self.__n_changesets = len(system.get("changesets"))
+                self.__refresh()
```

Comment:
to avoid the nested if's here I'd suggest something like 

```suggestion
        num_changesets = len(system.get("changesets",[]))
        if self.__n_changesets != num_changesets:
            self.__n_changesets = num_changesets
            self.__refresh()
```

Reply:
If you'd like the try the walrus out something like this works too 😃 

```suggestion
        if (num_changesets := len(system.get("changesets",[]))) != self.__n_changesets:
            self.__n_changesets = num_changesets
            self.__refresh()
```